packages/dds/tree/src/class-tree/schemaTypes.ts

```diff
@@ -320,7 +320,7 @@ export type NodeBuilderData<T extends TreeNodeSchema> = T extends TreeNodeSchema
  *
  * @public
  */
-export interface TreeMapNode<T extends ImplicitAllowedTypes>
+export interface TreeMapNode<T extends ImplicitAllowedTypes = ImplicitAllowedTypes>
```

Comment:
what does this do?

Reply:
There was some code that just wanted to refer to TreeMapNode in general without a specific schema, and this provides an easy way to do that via `TreeMapNode` with no generic arguments.